DevCycle/Models/PlatformDetails.swift

```diff
@@ -16,6 +16,12 @@ struct PlatformDetails {
     var deviceModel: String { UIDevice.current.model }
     var systemVersion: String { UIDevice.current.systemVersion }
     var systemName: String { UIDevice.current.systemName }
+    #elseif os(OSX)
+    
+    // TODO: figure out how to get this from macos: https://stackoverflow.com/questions/20070333/obtain-model-identifier-string-on-os-x
```

Comment:
need to implement

Reply:
https://github.com/DevCycleHQ/ios-client-sdk/pull/132